modules/mobile/app/services/mobile/v0/lighthouse_letters/service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module LighthouseLetters
+      class Service < Common::Client::Base
+        configuration Configuration
+
+        # rubocop:disable Lint/MissingSuper
+        def initialize(user)
+          @user = user
+          raise Common::Exceptions::BackendServiceException, 'no ICN associated with user' if user&.icn.blank?
```

Comment:
You're using the safety operator here. Do you think it's possible for user to be nil? I don't see how that would happen. If it somehow were to happen, I'm not sure the message "no ICN associated with user" is correct because there is no user.

Reply:
that's a good point. no I don't see that happening, I've removed the safety operator